### DISCUSSION TITLE
Toggle verticalAlignment

### DISCUSSION THREAD

**Discussion by clinvest** [filamentphp/filament#10853] [Help] [answered]
Hi all,

Is there a way to change the verticalAlignment of a toggle? I would like to position the field at the "end" because for now the render is strange.

<img width="1002" alt="Capture d’écran 2024-01-16 à 21 02 46" src="https://github.com/filamentphp/filament/assets/144620798/9d82d768-ba8c-4415-b4c0-fc0172db7fb9">

**Comment by ArtMin96** [comment, marked as answer]
```php
->inline(false)
```

**Reply by clinvest** [reply]
Yes I ended up with that thank you
